authui/src/authflowv2/botprotection/cloudflareTurnstile.ts

```diff
@@ -0,0 +1,43 @@
+import { Controller } from "@hotwired/stimulus";
+import { getColorScheme } from "../../getColorScheme";
+
+function parseTheme(theme: string): Turnstile.Theme {
+  switch (theme) {
+    case "light":
+      return "light";
+    case "dark":
+      return "dark";
+    case "auto":
+      return "auto";
+    default:
+      return "auto";
+  }
+}
+
+export class CloudflareTurnstileController extends Controller {
+  static values = {
+    siteKey: { type: String },
+  };
+
+  static targets = ["widget", "tokenInput"];
+
+  declare siteKeyValue: string;
+  declare widgetTarget: HTMLDivElement;
+  declare tokenInputTargets: HTMLInputElement[];
```

Comment:
Why do we need to support multiple inputs?

I am asking this question because you did not put a comment explaining the intended usage of this controller.

---

One possible approach to structure controllers
1. A controller for integrating the bot protection provider. Once verification succeeds, it can fire a custom event to broadcast the response.
2. A controller to be mounted on every `form` that could have captcha. This controller listen for the above custom event and set the input value.